code-challenge/generate-react-cli.json

```diff
@@ -0,0 +1,15 @@
+{
+  "usesTypeScript": true,
+  "usesCssModule": false,
+  "cssPreprocessor": "scss",
+  "testLibrary": "Testing Library",
+  "component": {
+    "default": {
+      "path": "src/components",
+      "withStyle": true,
+      "withTest": true,
+      "withStory": false,
+      "withLazy": true
+    }
+  }
+}
```

Comment:
A few years back, I attempted to make something similar to this because I wanted to make it easier for folks to create standardized components. I did not know that this existed! Thanks for the introduction 🙌🏾

Reply:
One thing I've liked about Angular (and it reminded me of Rails when I first encountered it) was the component generator. One thing I wish this library had was the concept of a service or interface (maybe it does, I just tried it for the first time last night).